CI/README.md

```diff
@@ -0,0 +1,32 @@
+## CI Tests
+
+### First steps
+Edit [my_tests](tests/my_tests) with tests you want to run
+
+### How to run
+```./CI/run.sh```
+
+This will run kraken using python, make sure python3 is set up and configured properly with all requirements
+
+
+### Adding a test case
```

Comment:
Maybe reference this section in https://github.com/cloud-bulldozer/kraken/blob/master/README.md#contributionsto with a note around tweaking the kraken/scenario configs in case of any changes to make sure users make changes for the CI to pass?

Reply:
Good idea, added